target/linux/mpc85xx/image/Makefile

```diff
@@ -16,15 +16,28 @@ define Device/Default
   KERNEL := kernel-bin
 endef
 
-define Build/tplink-v1-image
-	$(STAGING_DIR_HOST)/bin/mktplinkfw \
-		-H $(TPLINK_HWID) -W $(TPLINK_HWREV) -F $(TPLINK_FLASHLAYOUT) \
-		-N "$(VERSION_DIST)" -V $(REVISION) \
+define rootfs_align
```

Comment:
please move both helpers to the top of the Makefile. Place it below the
DEVICE_VARS += TPLINK_HWID TPLINK_HWREV TPLINK_FLASHLAYOUT

but before the define Device/Default.

in the commit message: intruduce -> introduce 

This looks really good.

Reply:
> please move both helpers to the top of the Makefile. Place it below the
>DEVICE_VARS += TPLINK_HWID TPLINK_HWREV TPLINK_FLASHLAYOUT
>
>but before the define Device/Default.

Done.

> in the commit message: intruduce -> introduce

Sorry.

> This looks really good.

Thank You.